data/en/alerts.yml

```diff
@@ -1,8 +1,8 @@
 ############################# Ensign Data ###################################
 notice:
-  enable: false
-  notice: "Planning & Scoping LLM Projects"
+  enable: true
+  notice: "Why ChatGPT Works as a Product — A Model is Not a Solution!"
   action:
     text: Register Now
-    link: "https://r8l.co/0Y7E5fFJ3ts"
-  expires: 2024-03-26T16:00:00Z
\ No newline at end of file
+    link: "https://us06web.zoom.us/webinar/register/5017139819706/WN_mijTPkO6Tr6HOCuqOCmlbw"
```

Comment:
I assume this is where you'll add the r8l.co short link?

Reply:
You are correct!